packages/@o3r/analytics/package.json

```diff
@@ -30,12 +30,16 @@
     "@ngrx/store": "~16.0.1",
     "@o3r/core": "workspace:^",
     "@o3r/dev-tools": "workspace:^",
+    "@o3r/schematics": "workspace:^",
```

Comment:
it's a not related bug ? :D

Reply:
it is not but is was a warning at post installed when I installed the dep before the PR, so I fixed it